docs/manage-cloud/cloud-data-management.md

```diff
@@ -0,0 +1,87 @@
+---
+description: How to manage your data on Cloud.
+contentType: howto
+---
+
+# Cloud data management
+
+There are two concerns when managing data on Cloud:
+
+* Memory usage: complex workflows processing large amounts of data can exceed n8n's memory limits. If this happens, the instance can crash and become inaccessible.
+* Data storage: depending on your execution settings and volume, your n8n database can grow in size and run out of storage. 
+
+To avoid these issues, n8n recommends that you build your workflows with memory efficiency in mind, and don't save unnecessary data
+
+## Memory limits on each Cloud plan
+
+Current plans:
+
+* Trial: 320MiB RAM, 10 millicore CPU burstable
+* Starter: 320MiB RAM, 10 millicore CPU burstable
+* Pro-1 (10k executions): 640MiB RAM, 20 millicore CPU burstable
+* Pro-2 (50k executions): 1280MiB RAM, 80 millicore CPU burstable
+* Enterprise: 4096MiB RAM, 80 millicore CPU burstable
+
+Legacy plans:
+
+* Start: 320MiB RAM, 10 millicore CPU burstable
+* Power: 1280MiB RAM, 80 millicore CPU burstable
+
+## How to reduce memory consumption in your workflow
+
+The way you build workflows affects how much data they consume when executed. Although these guidelines aren't applicable to all cases, they provide a baseline of best practices to avoid ex of instance memory.
+
+* Split the data processed into smaller chunks. For example, instead of fetching 10,000 rows with each execution, process 200 rows with each execution.
+* Avoid using the Code node where possible.
```

Comment:
Do we really want to say this? As we focus on more technical users, telling them not to use an essential core node is not ideal. How big of a performance hit is it?